fastlane/lib/fastlane/actions/appetize.rb

```diff
@@ -68,17 +69,15 @@ def self.parse_response(response)
         body = JSON.parse(response.body)
         app_url = body['appURL']
         manage_url = body['manageURL']
-        private_key = body['privateKey']
         public_key = body['publicKey']
 
-        Actions.lane_context[SharedValues::APPETIZE_PRIVATE_KEY] = private_key
         Actions.lane_context[SharedValues::APPETIZE_PUBLIC_KEY] = public_key
         Actions.lane_context[SharedValues::APPETIZE_APP_URL] = app_url
         Actions.lane_context[SharedValues::APPETIZE_MANAGE_URL] = manage_url
         return true
-      rescue
-        UI.error("Error uploading to Appetize.io: #{response.body}")
-        return false
+      rescue => ex
+        UI.error ex
+        UI.user_error!("Error uploading to Appetize.io: #{response.body}")
```

Comment:
What is the behavior of calling `UI.error ex`? I think UI.error is expecting a message. Will this just get the message from the exception?